index.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import cookieParser from 'cookie-parser';
+import cors from 'cors';
+import loginRoutes from './routes/login.js';
+import projectsRoutes from './routes/projects.js';
+
+const app = express();
+const PORT = 3030;
+
+app.use(cors());
+app.use(cookieParser());
+app.use(bodyParser.json());
+
+app.use('/login', loginRoutes);
+app.use('/projects', projectsRoutes);
+
+app.listen(PORT, () => console.log(`Server running on port http://localhost:${PORT}`));

```

Comment:
Nitpick but use `http.createServer`. Express was designed to pass it there, read up, and tell me why,

Reply:
Still not fixed.